test/expected/drop_extension.out

```diff
@@ -71,7 +71,25 @@ SELECT * FROM drop_test;
 
 --test drops thru cascades of other objects
 \c :TEST_DBNAME :ROLE_SUPERUSER
-drop schema public cascade;
+-- Stop background workers to prevent them from interfering with the drop public schema
+SELECT _timescaledb_functions.stop_background_workers();
+ stop_background_workers 
+-------------------------
+ t
+(1 row)
+
+DO
+$$
+BEGIN
```

Comment:
I'm sure this would help somewhat but from what I have seen, its not enough for complete certainty. 

I suggest we also revoke connections so that we are more secure to avoid flakiness, like here:
https://github.com/timescale/timescaledb/blob/main/test/sql/pg_dump.sql#L236